File: Arrays & Strings/arr.java
import java.util.Scanner;

public class arr {
    public static void main(String[] args) {
        // Simple technique to reverse print array
        Scanner scan = new Scanner(System.in);
        short n;
        n = scan.nextShort();
        short[] arr = new short[n];
        for (int i = 0; i < n; i++) {
            arr[i] = scan.nextShort();
        }
        for (int i = n - 1; i >= 0; i--) {
            System.out.print("|" + arr[i]);
        }
        scan.close();
    }
}

File: Arrays & Strings/arr.cpp
#include <iostream>
using namespace std;
int main()
{ // Reverse an array in numbers
    int n;
    cout << "Array size ?";
    cin >> n;
    int arr[n];
    for (int i = 0; i < n; i++)
    {
        cin >> arr[i];
    }
    for (int i = 0; i < n; i++)
    {
        cout << " " << arr[i];
    }
    return 0;
}